examples/ui-demo/src/app/page.tsx

```diff
@@ -8,11 +8,9 @@ import { AuthCardWrapper } from "../components/preview/AuthCardWrapper";
 import { CodePreview } from "../components/preview/CodePreview";
 import { CodePreviewSwitch } from "../components/shared/CodePreviewSwitch";
 import { TopNav } from "../components/topnav/TopNav";
-import { UserConnectionAvatarWithPopover } from "@/components/shared/user-connection-avatar/UserConnectionAvatarWithPopover";
 import { useUser } from "@account-kit/react";
-import { useConfig } from "./state";
+import { RenderUserConnectionAvatar } from "@/components/shared/user-connection-avatar/RenderUserConnectionAvatar";
 import { MobileSplashPage } from "@/components/preview/MobileSplashPage";
-import { useLogout } from "@account-kit/react";
 import { cn } from "@/lib/utils";
```

Comment:
Not a comment on this PR specifically, but this `cn` thing is the `clsx` equivalent I was thinking of. TBH I'm not sure why it exists in the first place because we have `clsx` added as a dependency, so we should just use that when we have conditionals in our class names. cc @RobChangCA